<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskLoadDlg</class>
 <widget class="QDialog" name="TaskLoadDlg">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>730</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>781</width>
     <height>571</height>
    </rect>
   </property>
   <property name="title">
    <string>文件列表</string>
   </property>
   <widget class="QTableWidget" name="tableWidgetData">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>30</y>
      <width>751</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>650</y>
     <width>781</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>操作</string>
   </property>
   <widget class="QPushButton" name="btnQuit">
    <property name="geometry">
     <rect>
      <x>688</x>
      <y>26</y>
      <width>84</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(232, 232, 232);</string>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOpen">
    <property name="geometry">
     <rect>
      <x>518</x>
      <y>26</y>
      <width>84</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(232, 232, 232);</string>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDel">
    <property name="geometry">
     <rect>
      <x>603</x>
      <y>26</y>
      <width>84</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(232, 232, 232);</string>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>27</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>781</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>路径</string>
   </property>
   <widget class="QRadioButton" name="radioButtonDefaultPath">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>40</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>默认路径</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonUDisk">
    <property name="geometry">
     <rect>
      <x>143</x>
      <y>40</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>U盘</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
